spaceship/lib/spaceship/tunes/tunes_client.rb

```diff
@@ -567,10 +567,10 @@ def update_member_roles!(member, roles: [], apps: [])
     # @!group AppAnalytics
     #####################################################
 
-    def time_series_analytics(app_ids, measures, start_time, end_time, frequency, view_by)
+    def time_series_analytics(app_ids, dimensions, measures, start_time, end_time, frequency, view_by)
```

Comment:
```suggestion
    def time_series_analytics(app_ids, measures, start_time, end_time, frequency, view_by, dimensions = [])
```

Move `dimensions` to the end and default it to an empty array

I would do something like this to allow backwards compatibility ☝️